node/resolvers/catalog/index.ts

```diff
@@ -265,10 +312,78 @@ export const queries = {
     return catalog.products(args)
   },
 
+  productsByIdentifier: async (
+    _: any,
+    args: ProductsByIdentifierArgs,
+    ctx: Context
+  ) => {
+    const {
+      clients: { catalog },
+    } = ctx
+
+    let products = [] as Product[]
+    let product = [] as Product[]
+    const { field, values } = args
+
+    switch (field) {
+      case 'id':
+        const idStart = async () => {
+          await asyncForEach(values, async (value: string) => {
+            product = await catalog.productById(value)
```

Comment:
is this calling the catalog API once for each value? how many values do you expect for this query to send at once? this seems to have the potential for being throttled/overloading the API I think

Reply:
Suppose we limited it to 20 values? 
As far as I'm aware, there's not a way to do a single API call that includes multiple identifier values. I'll check with Marcelo and make sure, though.

Reply:
@polishq I am checking this as we speak

Reply:
Never mind, I've figured out a way. I'll make some changes and recommit

Reply:
@polishq You can concatenate your filters like this:

`/pub/products/search?fq=alternateIds_Ean:1&fq=alternateIds_Ean:2&fq=alternateIds_Ean:3`

You can do this until the query reach `7000 characters` (the API does not support more than this). This will perform an `||` (or) on the filters and return the matched results.

Reply:
Maybe the best alternative is to perform a paginated query using the example above.

Reply:
See new commit